### PR TITLE
HIU-971 | adds auth/on-init and auth/on-confirm api for logging information

### PR DESCRIPTION
**Pull request by avghadsangita**

### CODE REVIEW

File: src/main/java/in/org/projecteka/hiu/user/model/GatewayResponse.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import lombok.Data;
+import lombok.Getter;
+
+@Builder
+@Getter
+@AllArgsConstructor
+@Data
+public class GatewayResponse {
+    private String requestId;
+}
```
Comment (dheeraj-p) [inline src/main/java/in/org/projecteka/hiu/user/model/GatewayResponse.java]: This model is already create somewhere. Please check.

File: src/main/java/in/org/projecteka/hiu/user/model/RespError.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@AllArgsConstructor
+@Builder
+@NoArgsConstructor
+@Data
+public class RespError implements Serializable {
+    private Integer code;
+    private String message;
+}
```
Comment (dheeraj-p) [inline src/main/java/in/org/projecteka/hiu/user/model/RespError.java]: This also is created somewhere.

File: src/main/java/in/org/projecteka/hiu/user/model/GatewayResponse.java (hunk 1 above)
Comment (avghadsangita) [reply src/main/java/in/org/projecteka/hiu/user/model/GatewayResponse.java]: done

File: src/main/java/in/org/projecteka/hiu/user/model/RespError.java (hunk 2 above)
Comment (avghadsangita) [reply src/main/java/in/org/projecteka/hiu/user/model/RespError.java]: done

**Pull request merged**